src/components/tool/guide/Instructions/StartInstructions.js

```diff
@@ -36,6 +37,16 @@ export function LyraStartInstructions({
     jobType !== "Interactive" && jobName !== "" ? ' -N "' + jobName + '"' : "",
   ];
 
+  const wallTimeStr =
+    jobType === "Interactive"
+      ? "01:00:00"
+      : (wallTime?.hour < 10 ? "0" : "") +
```

Comment:
FYI there is a `padStart` method on strings for this, but this is fine

Reply:
I could not find `padStart` method you mentioned in this project

Reply:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/padStart